### PR TITLE
Individual Icon import option

### PR DESCRIPTION
**Pull request by afzaalb**
I've added all the icons as individual components and they can now be imported as 
`import ArrowLeft from 'react-bytesize-icons/lib/components/ArrowLeft'`.

We can still import icons as we used to do previously, but the reason for doing this is when i'm only using 10 icons from a set of 94 icons my JS bundle increases significantly. 
The icons bundle is almost 52K.Technically i should be importing those specific icons so i import few kb's from icon-bundle instead of whole 52K. 
Hope my explanation is clear.
You can see the difference with both approaches in screenshot attached.

screenshot 2018-12-15 at 9 30 32 pm

### CODE REVIEW

**Comment by Kilian** [comment]
Hi @afzaalb this is great! I have one requests though: make sure the import path is short, so it should be `import Info from "react-bytesize-icons/info";` as that's much more discoverable. Can you also add a little to the README.md under 'how to use'?

**Comment by afzaalb** [comment]
Icons can now be imported as `import Twitter from react-bytesize-icons/Twitter`.I've also updated the readme and added a link to this PR.  @Kilian

File: package.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
   "name": "react-bytesize-icons",
   "version": "0.8.3",
   "description": "Bytesize Icons as React Components.",
-  "main": "lib/index.js",
```
[the remark this answers is not in the document]
Comment (afzaalb) [reply package.json]: so this means that i should mention the files array in package.json ? i didn't get what you're trying to say here !

[the remark this answers is not in the document]
Comment (Kilian) [reply package.json]: oh, my mistake! We already have an npmignore that makes sure we don't also publish the src and icons folder so this is all good.

**Comment by Kilian** [comment]
I will make a new release in the next few days, Thanks again @afzaalb!

**Comment by afzaalb** [comment]
Thank You !

**Pull request merged**